### PR TITLE
Add prettier

### PR DESCRIPTION
**Pull request by offerijns**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 language: node_js
-node_js:
-  - 12.16
```
Comment (philipstanislaus) [inline .travis.yml:3]: Why did we remove this?

File: package.json [hunk 2]
```diff
@@ -16,11 +16,13 @@
...
     "lint-fix": "tslint --fix --project .",
-    "lint-fix-staged": "git diff --staged --name-only | grep -E '\\.tsx?$' | xargs npm run lint-fix && git diff --staged --name-only | grep -E '\\.tsx?$' | xargs git add"
+    "lint-fix-staged": "git diff --staged --name-only | grep -E '\\.tsx?$' | xargs npm run lint-fix && git diff --staged --name-only | grep -E '\\.tsx?$' | xargs git add",
+    "format": "prettier --write \"./**/*.{ts,tsx}\"",
+    "check-format": "prettier --check \"./**/*.{ts,tsx}\""
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run lint-fix-staged && npm run build-for-husky"
+      "pre-commit": "npm run lint-fix-staged && npm run format && git add"
```
Comment (philipstanislaus) [inline package.json:25]: I'd like not to use `git add` here. If you have selectively staged things for this commit, that messes it up.

File: .travis.yml (hunk 1 above)
Comment (offerijns) [reply .travis.yml:3]: TravisCI automatically picks up `.nvmrc` now, so we don't need to specify it here.

File: package.json (hunk 2 above)
Comment (offerijns) [reply package.json]: Removed!

**Pull request merged**